test/integration/lib/make_suite.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+const Config = require('config-js');
+
+var config = new Config('./test/integration/lib/config/foxbox.js');
+var philipshue_server = require('../lib/philipsHue_server.js');
+var ipcamera_server = require('../lib/ipcamera_server.js');
+var webPush_server = require('../lib/webpush_server.js');
+var nupnp_server = require('../lib/nupnp_PhilipsHue.js');
+var foxboxManager = require('../lib/foxbox_process_manager.js');
+var config = new Config('./test/integration/lib/config/foxbox.js');
+
+
+var make_suite = (function() {
+  var hueSimOn = false;
+  var cameraSimOn = false;
+  var webPushSimOn = false;
+  
+  function turnAllSims() { 
```

Comment:
Nit: `on` is missing. I'd also explicitly name `simulators` instead of `sims` => `turnOnAllSimulators`